Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/submit_comment_box.tsx

```diff
@@ -0,0 +1,122 @@
+import * as Dali from 'dali';
+import * as React from 'react';
+import { CommentBox } from '../comment_box';
+import { DisplaySize } from '../../..';
+import { SubmitButton } from '.';
+
+interface Properties  {
+  displaySize: DisplaySize;
+  isSubmitEnabled: boolean;
+  submitStatus: string;
+  hasError: boolean;
+  onSubmit: () => void;
+}
+
+export class SubmitCommentBox extends React.Component<Properties > {
+  public static readonly defaultProps = {
+    isSubmitEnabled: false,
```

Comment:
Only provide ```defaultProps``` for optional props. If the props are not optional and will always be set/initialized then remove them from here.